File: revista-digi-app/src/modelo/Revista/comentario.ts
export class Comentario {

      contenido:String;
      usuario:String;
      revista:String;
      fecha:String;
      fechaD!:String;

      constructor(contenido:String, usuario:String, revista:String,fecha:String ){
        this.contenido= contenido;
        this.usuario = usuario;
        this.revista = revista;
        this.fecha = fecha;
      }


}

File: RevistasDigitales/src/main/java/com/proyectoipc/reports/entidades/Comentario.java

package com.proyectoipc.reports.entidades;

import java.util.Date;

/**
 *
 * @author elvis_agui
 */
public class Comentario {
    private String usuario;
    private String contenido;
    private String revista;
    private Date fecha;

    public String getUsuario() {
        return usuario;
    }

    public void setUsuario(String usuario) {
        this.usuario = usuario;
    }

    public String getContenido() {
        return contenido;
    }

    public void setContenido(String contenido) {
        this.contenido = contenido;
    }

    public String getRevista() {
        return revista;
    }

    public void setRevista(String revista) {
        this.revista = revista;
    }

    public Date getFecha() {
        return fecha;
    }

    public void setFecha(Date fecha) {
        this.fecha = fecha;
    }
    
    
}
